deploy/clowdapp.yml

```diff
@@ -81,7 +81,9 @@ objects:
                   strip_path_prefix: "/api/rhsm"
                 - handler: rbac
                   service_url: http://{{(index .Deps "rbac" "service").Hostname}}:{{(index .Deps "rbac" "service").Port}}/api/rbac/v1
-                  application: subscriptions
+                  application: 
```

Comment:
Same comment as above. Since this is already changing once, it might make sense to make this a template parameter. See how `APP_NAME` is configured as a reference if you choose to go that route.